collections-csharp/IterateThroughDictionary/Tests/DictionaryTest.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using IterateThroughDictionary;
+using System.Collections.Generic;
+using System.IO;
+
+namespace DictionaryTests
+{
+    [TestClass]
+    public class DictionaryTest
+    {
+        public static readonly string _monthJanuary = "1 : January";
+        public static readonly string _monthFebruary = "2 : February";
+        public static readonly string _monthMarch = "3 : March";
+        public static readonly string _monthApril = "4 : April";
+        public static readonly string _monthJanuaryStringJoin = "[1, January]";
+
+        StringWriter _stringWriter = new StringWriter();

```

Comment:
Make sure to explicitly state public/private/protected for consistency.

Reply:
variable made private